### ISSUE TITLE
Getting error for recusive apk fetch for bash (Alpine 3.7.0)

### ISSUE THREAD

**Issue by oliverorav** [gliderlabs/docker-alpine#383]
Unable to do apk fetch with dependencies for bash (and some other like coreutils). 
Getting following error:
```
# apk fetch -R -v bash
bash: unable to select package (or its dependencies)
ERROR: unsatisfiable constraints:
  /bin/sh (virtual):
    provided by: busybox
    required by: bash-4.4.19-r1[/bin/sh] bash-4.4.19-r1[/bin/sh] bash-4.4.19-r1[/bin/sh]
```
Without -R seems to be working and also same command in 3.6 is working.

**Comment by kaniini** [comment]
This is fixed in the apk-tools 2.9.1 package I backported to 3.7.

**Comment by oliverorav** [comment]
Yes, seems to be working now. Thank you!
